README.md

```diff
@@ -44,6 +46,14 @@ The dependencies of the docsy theme can be installed as follows:
 npm ci
 ```
 
+A [flox](https://flox.dev/docs/install-flox/) environment is
+also available and can be activated with:
```

Comment:
Maybe it should got a dedicated sub section and contain a few words about what/why flox.